src/mod/codecs/mod_opus/mod_opus.c

```diff
@@ -1160,6 +1168,18 @@ static switch_status_t switch_opus_keep_fec_enabled(switch_codec_t *codec)
 	}
 }
 
+static switch_bool_t switch_opus_vad(struct opus_context *context, void *encoded_data, uint32_t encoded_data_len) {
+	const uint8_t *payload = (const uint8_t *) encoded_data;
+	packet_context_t packet_context=SWITCH_OPUS_PKT_RX;
+	switch_opus_packet_extract_data(payload, encoded_data_len, &context->control_state_decode.opus_packet_info, &context->control_state_decode.rtp_codec_data, packet_context);
+	switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "OPUS EXTRACT PAYLOAD VAD len:%d vad_ms:%d bytes:%02x %02x\n",
```

Comment:
can we print this only when "if (globals.debug || context->debug > 1) {" , if it's in line with the other per-packet debug logs. also, I think switch_opus_info() should show if VAD here, just like the other characteristics:
```
switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "%s: opus_frames [%d] samples [%d] audio bandwidth [%s] bytes [%d] FEC[%s] channels[%d]\n",
 print_text, nb_opus_frames, nb_samples, audiobandwidth_str, len, has_fec ? "yes" : "no", nb_channels);
```